Plots/Meteograms/NCL_meteo_1.py

```diff
@@ -0,0 +1,291 @@
+                       levels=bounds,
+                       linewidths=0.1,
+                       zorder=3)
+
+# Plot contours for the tempisobar variable
+contour3 = ax1.contour(tempisobar,
+                       transform=ccrs.PlateCarree(),
+                       colors='red',
+                       levels=[-20, -10, 0, 10, 20, 30, 40, 50, 60],
+                       linewidths=0.4,
+                       linestyles='solid',
+                       zorder=4)
+
+# Create lists of coordinates where the contour labels are going to go
+cont2Labels = [(1.71, 3.82), (5.49, 3.23), (9.53, 4.34), (9.27, 3.53),
+               (14.08, 4.81), (19.21, 2.24), (17.74, 1.00), (22.23, 3.87),
+               (12.87, 2.54), (10.45, 6.02), (11.51, 4.92)]
+
+cont3Labels = [(7.5, 6.1), (10.0, 4.58), (19.06, 1.91), (8.68, 0.46),
+               (19.52, 4.80), (16.7, 6.07), (8.62, 5.41), (18.53, 5.46)]
+
+# Manually plot contour labels
+cont2labels = ax1.clabel(contour2,
+                         manual=cont2Labels,
+                         fmt='%d',
+                         inline=True,
+                         fontsize=7)
+cont3labels = ax1.clabel(contour3,
+                         manual=cont3Labels,
+                         fmt='%d',
+                         inline=True,
+                         fontsize=7,
+                         colors='k')
+
+# Set contour label backgrounds white
+[txt.set_bbox(dict(facecolor='white',
+                   edgecolor='none',
+                   pad=.5)) for txt in contour2.labelTexts]
+[txt.set_bbox(dict(facecolor='white',
+                   edgecolor='none',
+                   pad=.5)) for txt in contour3.labelTexts]
+
+# Use the geocat.viz function to set the main title of the plot
+gvutil.set_titles_and_labels(ax1,
+                             maintitle='Meteogram for LGSA, 28/12Z',
+                             maintitlefontsize=18,
+                             ylabel='Pressure (mb)',
+                             labelfontsize=12)
+
+# Determine the labels for each tick on the x and y axes
+yticklabels = np.array(levels, dtype=np.int)
+xticklabels = ['12z', '15z', '18z', '21z', 'Apr29',
+               '03z', '06z', '09z', '12z', '15z', '18z',
+               '21z', 'Apr30', '03z', '06z', '09z',
+               '12z', '15z', '18z', '21z', 'May01', '03z',
+               '06z', '09z', '12z']
+
+# Use the geocat.viz function to set axes limits and ticks
+gvutil.set_axes_limits_and_ticks(ax1,
+                                 xlim=[0, 24],
+                                 ylim=[0, 7],
+                                 xticks=np.arange(0, 25),
+                                 yticks=np.arange(0, 8),
+                                 xticklabels=xticklabels,
+                                 yticklabels=yticklabels)
+
+# Make the ticks on each axis point inwards instead of outwards
+ax1.tick_params(axis="x", direction="in")
+ax1.tick_params(axis="y", direction="in", labelsize=9)
```

Comment:
Your plot doesn't have inward facing ticks. I don't know if these two lines did what you expected them to do.